exercises/resistor-color-duo/.meta/src/reference/kotlin/ResistorColorDuo.kt

```diff
@@ -0,0 +1,6 @@
+object ResistorColorDuo {
+
+    private val colors = listOf("black", "brown", "red", "orange", "yellow", "green", "blue", "violet", "grey", "white")
```

Comment:
Can we invent an **enum** here, please? )  It'd become a part of public api and that would follow the general habits of the track implementations.

Reply:
ok, can do :)
Should the tests use the constants from Colors enum or continue to call value() with strings?
The C track uses enum constants but defines them only in the example.h whereas the Java track uses string parameters and keeps Colors a `private enum` in the example solution.

Reply:
Here are lots of samples in our existing sourcebase, and you've recently implemented `Yacht` exercise with the same pattern (have a look at `YachtCategory` to recall):
 - define `enum class` in `src/main/kotlin` next to the stub implementation and thus this enum becomes a part of public api and can be used in signatures like 
```kotlin
 enum class Color ...

 fun value(vararg colors: Color): Int
```
 - place the same-named *link*-file next to reference implementaction 
 - use the enum both in ref implementation and test suite. I suppose, we must not insist on lower case, but should explore enum constants directly.

Does this sound reasonable?

Reply:
The challenge feels almost too easy now :)